v4l2-request-test.c

```diff
@@ -257,11 +258,34 @@ int main(int argc, char *argv[])
 	setup_config(&config);
 
 	while (1) {
-		opt = getopt(argc, argv, "v:m:d:D:s:f:P:ilqh");
+	        int option_index = 0;
+		static struct option long_options[] = {
```

Comment:
Why declare it as static here? This is almost certainly not what you want to do.